sdk/keyvault/azure-security-keyvault-secrets/CMakeLists.txt

```diff
@@ -31,6 +31,18 @@ include(AzureConfigRTTI)
 include(AzureBuildTargetForCI)
 # Add create_map_file function
 include(CreateMapFile)
+option(TSP_GEN "Generate from TypeSpec" OFF)
+
+if(TSP_GEN)
+    include(TSPCompile) 
+
+    set(TSP_DESTINATION "keyvault_secrets")
+    set(TSP_REPO "https://github.com/Azure/azure-rest-api-specs.git")
+    set(TSP_BRANCH "mccoyp/kv-secrets-tsp")
+    set(TSP_REPO_PATH "specification/keyvault/data-plane/Security.KeyVault.Secrets/")
```

Comment:
Do we need repo and repo path? Can that be consolidated into one entry?

Reply:
kinda, repo is the repo url ( https://..../xyz.git) the path is the subfolder, they are also diffrently used in commands, thus avoiding extra string parsing in cmake.